test/mocks/stream_info/mocks.h

```diff
@@ -98,7 +102,10 @@ class MockStreamInfo : public StreamInfo {
   MOCK_METHOD(absl::optional<uint64_t>, upstreamConnectionId, (), (const));
   MOCK_METHOD(void, setAttemptCount, (uint32_t), ());
   MOCK_METHOD(absl::optional<uint32_t>, attemptCount, (), (const));
-
+  MOCK_METHOD(const BytesMetererSharedPtr&, getUpstreamBytesMeterer, (), (const));
```

Comment:
Can you try to use the same approach as the `connectionTerminationDetails` method (use the `ReturnPointee`), please, to see if fixes clang cl build error?

Reply:
Fixed.